sw/device/sca/aes_serial.c

```diff
@@ -20,6 +20,9 @@
  *   - Version ('v')+,
  *   - Seed PRNG ('s')+,
  *   - Batch encrypt ('b')*,
+ *   - FvsR batch fixed key set ('t')*,
+ *   - FvsR batch generate ('g')*,
+ *   - FvsR batch encrypt ('f')*,
```

Comment:
I think to align this with the latest suggestions in lowRISC/ot-sca#77, the "f" command should be called "FvsR batch encrypt and generate" and call the generate function **after** finishing the encrypt.